implementations/rust/ockam/ockam_command/src/terminal/fmt.rs

```diff
@@ -0,0 +1,39 @@
+#[macro_export]
+macro_rules! fmt_ok {
+    ($input:expr) => {
+        format!("{} {}", "✔".light_green(), format!($input))
+    };
+    ($input:expr, $($args:expr),+) => {
+        format!("{} {}", "✔".light_green(), format!($input, $($args),+))
+    };
+}
+
+#[macro_export]
+macro_rules! fmt_info {
+    ($input:expr) => {
+        format!("{} {}", ">".light_green(), format!($input))
+    };
+    ($input:expr, $($args:expr),+) => {
+        format!("{} {}", ">".light_green(), format!($input, $($args),+))
+    };
+}
+
+#[macro_export]
+macro_rules! fmt_warn {
+    ($input:expr) => {
+        format!("{} {}", "!".light_yellow(), format!($input))
+    };
+    ($input:expr, $($args:expr),+) => {
+        format!("{} {}", "!".light_yellow(), format!($input, $($args),+))
+    };
+}
+
+#[macro_export]
+macro_rules! fmt_err {
+    ($input:expr) => {
+        format!("{} {}", "×".light_red(), format!($input))
+    };
+    ($input:expr, $($args:expr),+) => {
+        format!("{} {}", "×".light_red(), format!($input, $($args),+))
+    };
+}
```

Comment:
I really like this :+1:

Reply:
And I found it pretty comfortable to use. With these macros + the use of `indoc` we can create all sorts of output structures (single/multi line) and with lint support.